eZ/Publish/API/Repository/RoleService.php

```diff
@@ -350,6 +355,18 @@ public function assignRoleToUser(Role $role, User $user, RoleLimitation $roleLim
     public function unassignRoleFromUser(Role $role, User $user);
 
     /**
+     * Loads a role assignment for the given id.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the authenticated user is not allowed to read this role
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException If the role assignment, role or user/group was not found
```

Comment:
`If the role assignment was not found` should be enough. Otherwise the data is not consistent and we don't assume that in the API.